src/routing.js

```diff
@@ -215,6 +216,8 @@ export function getPreferencesSearch({ section, subsection, data = {} } = {}) {
 
 const RoutingContext = React.createContext()
 
+let currentPath = ''
+
```

Comment:
same here, see if it's possible to detect the change of path without this global variable.

Reply:
ok thanks